client/src/app/components/mission-single/mission-single.component.ts

```diff
@@ -236,7 +240,7 @@ export class MissionSingleComponent implements OnInit {
         to: 'myclient@localhost.com',
         authors: this.users,
         state: 'draft',
-        scope: hosts,
+        scope: this.hosts,
```

Comment:
You use this.hosts instead of the host constant initialize during the map line 218

Reply:
thank you, that's why it didn't work.